edit-instrument-recordings-links.user.js

```diff
@@ -52,7 +52,7 @@ header.appendChild(headerColumn);
 // We add the links to the recordings
 for (let i = 0; i < recordings.length; i++) {
     const recordingRow = recordings[i];
-    const recordingUrl = recordingRow.childNodes[3].childNodes[0].getAttribute('href');
+    const recordingUrl = recordingRow.childNodes[1].childNodes[0].getAttribute('href');
```

Comment:
Isn't there a better way to find the url ?

Using jQuery, you could select links starting with `/recording/` (http://api.jquery.com/attribute-starts-with-selector/)
It would make it more resistant to future changes.